docs/pages/enterprise/sso/oidc.mdx

```diff
@@ -23,14 +23,37 @@ administrators to define policies like:
 ## Enable OIDC Authentication
 
 First, configure Teleport auth server to use OIDC authentication instead of the local
-user database. Update `/etc/teleport.yaml` as show below and restart the
-teleport daemon.
+user database. For self-hosted update `/etc/teleport.yaml` and restart the teleport daemon.  
+Use the Dynamic resources for Teleport Cloud and any other deployments.
```

Comment:
Let's change this to:

> You can use dynamic resources for Teleport Cloud as well as self-hosted deployments.

- It's the first mention, so "the" isn't really appropriate
- I think "self-hosted" is more precise than "any other" here, since there are only two deployment types
- Added "you can" to emphasize that this is optional for self-hosted deployments